backend/kafka-queue/types.go

```diff
@@ -37,6 +37,7 @@ const (
 	PushLogsFlattened                      PayloadType = iota
 	PushTracesFlattened                    PayloadType = iota
 	PushSessionEvents                      PayloadType = iota
+	PushSessionFields                      PayloadType = iota
```

Comment:
is there a difference between a `SessionEvent` and a `SessionField`? seems to me like the 2 can be consolidated